src/pyotp/__init__.py

```diff
@@ -11,8 +11,8 @@
 
 
 def random_base32(length=16, random=random.SystemRandom(),
-                  chars=base64._b32alphabet.values()):
+                  chars=list('ABCDEFGHIJKLMNOPQRSTUVWXYZ234567')):

```

Comment:
Because the base32 alphabet does not include digits '0', '1', '8' and '9'.
https://en.wikipedia.org/wiki/Base32#RFC_4648_Base32_alphabet